service/build.gradle

```diff
@@ -9,7 +9,16 @@ plugins {
 apply from: 'generators.gradle'
 apply from: 'publishing.gradle'
 
+repositories {
+	mavenCentral()
+	google()
+	maven {
+		url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
+	}
+}
+
 dependencies {
+	implementation  group: "com.azure.resourcemanager", name: "azure-resourcemanager-managedapplications", version: "1.0.0-beta.1"
```

Comment:
Normally we pull in cloud libraries through CRL. We still use the cloud libraries directly, but CRL is able to inject cleanup/logging for resource creations, etc.

BPM is not creating any cloud resources though, so I feel less strongly that it must go through CRL… so this seems ok I think.